### ISSUE TITLE
New English Terms needing Definitions

### ISSUE THREAD

**Issue by elletjies** [carpentries/glosario#1032]
- [ ] Conditional
- [ ] Array
- [ ] Stack
- [ ] Queue
- [ ] Binary Tree
- [ ] Heap
- [ ] Set
- [ ] Linked List
- [ ] Sorting
- [ ] Searching
- [ ] Dynamic Programming
- [ ] Greedy Algorithm
- [ ] Depth-First Search (DFS)
- [ ] Breadth-First Search (BFS)
- [ ] Hashing
- [ ] Backtracking
- [ ] Memoization
- [ ] Divide and Conquer
- [ ] Dataset
- [ ] Label
- [ ] Training Set
- [ ] Test Set
- [ ] Validation Set
- [ ] Data Cleaning
- [ ] Data Preprocessing
- [ ] Missing Data
- [ ] Probability
- [ ] Distribution
- [ ] p-value
- [ ] Hypothesis Testing
- [ ] Regression
- [ ] Dimensionality Reduction
- [ ] Principal Component Analysis (PCA)
- [ ] Underfitting
- [ ] Variance (ML Context)
- [ ] Feature Selection
- [ ] Model Training
- [ ] Model Evaluation
- [ ] Precision
- [ ] Recall
- [ ] F1 Score
- [ ] AUC (Area Under Curve)
- [ ] Random Forest
- [ ] Support Vector Machine (SVM)
- [ ] K-Nearest Neighbours (KNN)
- [ ] K-Means
- [ ] Naive Bayes
- [ ] Activation Function
- [ ] Recurrent Neural Network (RNN)
- [ ] LSTM (Long Short-Term Memory)
- [ ] Transformer
- [ ] Learning Rate
- [ ] Epoch
- [ ] Batch Size
- [ ] Loss Function
- [ ] TensorFlow

**Comment by olexandr-konovalov** [comment]
There are also some older issues with new terms suggestions: #722 #253 #239. The oldest is from 2020, I haven't checked systematically which of the older ones are already implemented, but I have seen at least a few (ticked "feature" and "ipython").
